### DISCUSSION TITLE
May I ask how to enable automatic video playback for the Windwos app SDK

### DISCUSSION THREAD

**Discussion by ShaoGongBra** [MicrosoftEdge/WebView2Feedback#4214] [Q&A] [answered]
I see that the answers are all about. net, and I don't know how to operate the Windwos app

**Comment by champnic** [comment, marked as answer]
Basic Windows App SDK info here: https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui

For setting command line args from the other issues you commented on, you can set an environment variable:
`Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "<additional args here>");`

**Reply by ShaoGongBra** [reply]
Thank you very much. This is indeed useful

**Reply by ShaoGongBra** [reply]
Specifically, execute this method before initializing WebView2
```c++
SetEnvironmentVariable(L"WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", L"--autoplay-policy=no-user-gesture-required");
```
